src/components/Search/SearchBar/index.tsx

```diff
@@ -90,6 +93,17 @@ const SearchBar: React.FC<SearchBarProps> = ({
     leaveBreadcrumb("search-bar-select", { value }, "user");
   };
 
+  const onKeyDown = (e: KeyboardEvent<HTMLInputElement>) => {
+    const commandKey = e.ctrlKey || e.metaKey;
+    if (e.key === "Enter" && commandKey && isValid) {
+      handleOnSubmit();
+    } else if (e.key === "Enter" && e.shiftKey) {
+      paginate(DIRECTION.PREVIOUS);
+    } else if (e.key === "Enter") {
+      paginate(DIRECTION.NEXT);
```

Comment:
you can use the `ignoreFocus` option in the original `useKeyboardShortcut` to allow pagination if the input is focused.

Reply:
No need to reimplement pagination here.

Reply:
I think it's important to tie pagination to the search bar explicitly, because we don't want to allow pagination from any text input:

https://user-images.githubusercontent.com/47064971/215210402-5b2caf0c-5dce-40b0-913d-56eb5210539c.mov

I do think it would be possible to delete the keyboard shortcuts for `Enter` and `Shift + Enter` though... I don't think they really needed under this new functionality.

Reply:
hmm good point. I don't like that the logic for pagination is tied to this input however since they are applying on separate functions. Would a wrapper component that encompasses both make sense? Or maybe refactor `useKeyboardShortcut` so that it accepts an `allowList` of accepted inputs.